cleanlab/datalab/internal/spurious_correlation.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import cross_val_score
+from sklearn.naive_bayes import GaussianNB
+from datalab import Datalab
+
+class SpuriousCorrelations:
+
+    def __init__(self, data: Datalab) -> None:
+        self.data = data
+        self.issues: pd.DataFrame = data.issues  
+        self.labels: np.ndarray = data.labels  
+        self.issue_summary: pd.DataFrame = data.issue_summary 
+
+    def spurious_correlations(self) -> pd.DataFrame:
+        baseline_accuracy = np.bincount(self.labels).argmax() / len(self.labels)
+        image_properties = ['outlier','near_duplicate','non_iid','low_information','dark','blurry','light','odd_aspect_ratio','odd_size']
+        property_scores = {}
+        for property_of_interest in image_properties:
+            if self.issue_summary[self.issue_summary['issue_type'] == property_of_interest]['num_issues'].values[0] > 0:
```

Comment:
I am guess this code will crash if `property_of_interest` does not exist amongst the issue types. You need to make sure code works even when some property_of_interest was never considered in the first place. 

the right way is before this for loop simply do:

```
image_properties_of_interest = ['outlier', 'near_duplicate', 'non_iid', 'low_information', 'dark', 'blurry', 'light',' odd_aspect_ratio', 'odd_size']
image_properties = [i for i in image_properties_of_interest if i in self.issue_summary['issue_type']]
```